h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -332,7 +332,9 @@ public Model getModel(MP params) {
    */
   private void appendFailedModelParameters(final Key<Model> modelKey, final MP params, final String[] rawParams,
                                            final Throwable t) {
-      final String failureDetails = isJobCanceled(t) ? "Job Canceled" : t.getMessage();
+      final String failureDetails = isJobCanceled(t) 
+              ? "Job Canceled, keeping the incomplete grid with all the models trained before cancellation." 
```

Comment:
this should describe individual model failure, it shouldn't have any relation to grid

Reply:
right, will revert